src/Libraries/CoreNodes/List.cs

```diff
@@ -423,8 +423,8 @@ public static object MaximumItem(IEnumerable<object> list)
         ///     except the first.
         /// </summary>
         /// <param name="list">List to be split.</param>
-        /// <returns name="first">First item in the list.</returns>
-        /// <returns name="rest">Rest of the list.</returns>
+        /// <returns name="first">type: var[]..[] (first item in the list)</returns>
```

Comment:
As above descriptions must come first followed by the type in brackets.

Reply:
Addressed in f347775